testing/test_hookcaller.py

```diff
@@ -187,6 +187,21 @@ def he_myhook1(arg1):
         assert not hasattr(he_myhook1, name)
 
 
+def test_hookimpl_specname():
+    """Make sure functions decorated with specname get the appropriate tag"""
+
+    @hookimpl()
+    def he_myhook1(arg1):
+        pass
+
+    @hookimpl(specname="name")
+    def he_myhook2(arg1):
+        pass
+
+    assert he_myhook1.example_impl.get("specname") is None
```

Comment:
Hmm not that it's a big deal but if `hooks.py`:130 was removed this check would still pass.
Not sure if we need to be strict about the "normalization" entry in the opts `dict` here.

Reply:
yeah good point. that's probably an unnecessary test, since `test_hookimpl` already pretty much makes sure that hookimpl parameters get added.  Was just trying to follow the pattern.  Should I just remove it?

Reply:
Yeah I'm not sure. If it's redundant I guess we don't need it?

Reply:
removed it